File: core/src/main/java/com/graqr/threshr/model/redsky/store/CapabilityHour.java
package com.graqr.threshr.model.redsky.store;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

import java.util.List;

@Serdeable
public record CapabilityHour(@JsonProperty("capability_code") String capabilityCode,
                             List<Day> days) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/store/CapabilityHour.kt
package com.graqr.threshr.circleStore.model.store

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class CapabilityHour(@param:JsonProperty("capability_code") val capabilityCode: String, val days: List<Day>) {}
